src/ocean/time/Time.d

```diff
@@ -23,7 +23,10 @@ module ocean.time.Time;
 import ocean.transition;
 import ocean.text.convert.DateTime_tango;
 
-version (UnitTest)
+// This causes circular references because `Test` imports `Layout_tango`
+// which imports this. Enable again when `Test` switches to the Formatter.
+// Formatter imports `Test` through `Traits`, so we cannot use `assert` either
```

Comment:
That's what I originally went with, but as the comment mention, `Formatter` imports `Test` too. So I cannot test `Formatter` from this module for this patch release.
However, starting from v2.6.x, `Test` uses `Formatter`. So you can just revert this on merging.